hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/contract/precompile/PrecompileFactory.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.web3.evm.store.contract.precompile;
+
+import com.hedera.services.store.contracts.precompile.Precompile;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import javax.inject.Named;
```

Comment:
Need to switch javax to jakarta everywhere after Spring Boot 3 PR.
```suggestion
import jakarta.inject.Named;
```

Reply:
@steven-sheehy Unfortunately, for the moment, the switch can't happen everywhere, since the `hedera-evm` library relies on the `javax.inject` package and on `mirror-node` side we initialize objects from the library using `javax.inject` arguments in their constructor, so if we switch mirror-node's usage to `jakarta` on these files, it's a breaking change.

I'm not sure what the consequences of this mix between javax.inject and jakarta might be.

Reply:
It's fine to have both javax and jakarta in web3 at the same time as long as Spring only loads beans via jakarta. You can keep hedera-evm with javax since we don't use component scanning to load any beans within evm library. It's okay to keep `EvmOperationConstructionUtil` returning `javax.inject.Provider`. But every Spring bean we create inside web3 module should be jakarta.

Reply:
It's working fine now on main with mixing javax and jakarta. I'm only asking that within your diff to change any places from `javax` to `jakarta`.